openstack/cinder/templates/vcenter_datacenter_cinder_deployment.yaml

```diff
@@ -49,6 +49,8 @@ template: |
         hostname: cinder-volume-vmware-{= name =}
         {{- tuple . "{= availability_zone =}" | include "utils.kubernetes_pod_az_affinity" | nindent 8 }}
         {{- include "utils.proxysql.pod_settings" . | nindent 8 }}
+        initContainers:
+        {{- tuple . (dict "service" (print "cinder-api," .Release.Name "-rabbitmq")) | include "utils.snippets.kubernetes_entrypoint_init_container" | indent 8 }}
```

Comment:
I'm not sure about this one. I looked at the `nova-compute` template and we don't have any dependencies defined. It was the same here before (no env to configure `kubernetes-entrypoint`, so no dependencies).

Why would `cinder-volume` depend on `cinder-api`? I think this should be `cinder-mariadb` instead?

Do we need to wait for the migration job to finish before starting up?

Reply:
> Why would cinder-volume depend on cinder-api? I think this should be cinder-mariadb instead?

I think, my idea was that `cinder-volume` talks to `cinder-api`  (via rabbitmq). It does not access the db directly, AFAIK.

> Do we need to wait for the migration job to finish before starting up?

Probably, I'll add it.

Reply:
Okay, it gets complicated... We probably do want to wait on a major release upgrade, but not just because some random attribute has changed.

Say, we change add a newline in the script, that triggers the creation of a new job, because that is an immutable attribute. But I don't think we want to replace the job right then.

I would say, it would make sense to revisit the migration jobs, but I would like to consider that a different problem then the one I am addressing here.

Reply:
`cinder-volume` does have a `proxysql` container in it and has the DB configure in `cinder.conf`. I also found none of the `remotable` decorators Nova has in the code.

Reply:
Ah, then we definitely want to wait on the db. I'll add it.

Reply:
> I would say, it would make sense to revisit the migration jobs, but I would like to consider that a different problem then the one I am addressing here.

follow-up somewhen later sounds fine. we didn't make things worse here, as there was no waiting done before anyways.